### DISCUSSION TITLE
Are we not allowed to use 😎 in vscode extension descriptions anymore?

### DISCUSSION THREAD

**Discussion by RandomFractals** [microsoft/vscode-discussions#107] [Extension Development QnA]
I republished my Data Table Renderers and JS Notebook Inspector extensions per requests from devs wanting to use them with Databricks & PowerBI notebooks and new Quarto Pub tools R devs are using, but our handy emoji tagging in extension descriptions now show up as random number of ??

![marketplace-question-marks](https://user-images.githubusercontent.com/656833/185623133-617f2728-ebe2-45cf-891e-1217ce8cb80d.png)

Not sure if it's worth logging it in vscode marketplace repo, or vscode repo itself. I'll leave that up to @isidorn and moderators here to decide.

Just wanted to see if others are having the same issues when republishing their old extensions, without any extension code or description modifications in `package.json`.

See github repos for what those extension descriptions should show up as (they did before):

https://github.com/RandomFractals/vscode-data-table/blob/main/package.json

https://github.com/RandomFractals/js-notebook-inspector/blob/master/package.json

**Comment by isidorn** [comment]
@prashantvc might know more here.

**Comment by prashantvc** [comment]
As far as I know we haven't changed anything regarding emojis. Let me verify this

**Comment by olguzzar** [comment]
It appears that the Data Table Rendered extension is no longer showing these ??? in the marketplace, so I am closing this discussion
